app/src/main/java/ca/chronofit/chrono/util/listeners/RepeatListener.kt

```diff
@@ -0,0 +1,66 @@
+package ca.chronofit.chrono.util.listeners
+
+import android.os.Handler
+import android.os.Looper
+import android.view.MotionEvent
+import android.view.View
+
+// Following: https://stackoverflow.com/questions/4284224/android-hold-button-to-repeat-action
+class RepeatListener(clickListener: View.OnClickListener) : View.OnTouchListener {
+    var handler = Handler(Looper.getMainLooper())
+    private var onClickListener: View.OnClickListener? = clickListener
+    private var touchView: View? = null
+    @Volatile
+    private var counter = 0
+
+    private val runnable = object : Runnable {
+        override fun run() {
+            if (touchView!!.isEnabled) {
+                if(counter > 3){
+                    handler.postDelayed(this, FAST_INCREMENT_PERIOD)
+                }else{
```

Comment:
Auto format please

Reply:
Addressed.
Decreased the time it takes to go from "slow rapid increment" to "fast rapid increment". LMK how you like this. I feel like if i decrease time between increments was any faster, the user will easily overshoot the time they want.